### PR TITLE
lectia-1: Add repsonive box style

### PR DESCRIPTION
**Pull request by 7ewen**

### CODE REVIEW

File: scss/layout/_section.scss [hunk 1]
```diff
@@ -1,25 +1,52 @@
...
+}
 
-// .section .section__title {
-//   border-bottom: 1px solid #000;
-// }
+.sidebar-area li {
+  padding-top: 16px;
+  padding-bottom: 13px;
+  padding-left: 22px;
+  border-bottom: #EEEEEE solid;
```
Comment (alexcracea) [inline scss/layout/_section.scss:41]: please add `1px` to the border to make it explicit here and everywhere you use border

File: scss/layout/_section.scss [hunk 2]
```diff
@@ -1,25 +1,52 @@
...
-// }
\ No newline at end of file
+.sidebar-area a {
+  color: #3B3D40;
+  text-decoration: none;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+}
```
Comment (alexcracea) [inline scss/layout/_section.scss:52]: Please leave an empty line here and generally at the end of every text file, see how github suggests it?

File: scss/layout/_section.scss [hunk 3]
```diff
@@ -1,25 +1,52 @@
...
-    margin-top: 30px;
-  }
+.section__title {
+  font-size: 56px;
+  line-height: 67px;
+  color: #333333;
 }
 
+.section__content {
+  padding-top:30px;
```
Comment (alexcracea) [inline scss/layout/_section.scss:23]: please add space after each colon symbol

File: scss/components/_header.scss [hunk 4]
```diff
@@ -1,49 +1,19 @@
...
-  line-height: 19px;
-  text-decoration: none;
+  justify-content: space-around; 
 }
 
-.button--white {
-  color: #fff;
-  border: solid #fff;
-  border-radius: 24px;
+.header input#mySearch {
```
Comment (alexcracea) [inline scss/components/_header.scss:12]: the selector is very specific, you don't need priority of 111 for this one, add a class to the text input to target it easier, or at least use the ID alone `#mySearch {}`

File: index.html [hunk 5]
```diff
@@ -4,215 +4,54 @@
...
-      </a>
-      <ul class="main-menu">
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Services</a></li>
-        <li><a href="#">Blog</a></li>
-      </ul>
-      <a href="#" class="button button--white">Sign In</a>
+    <input type="search" id="mySearch" placeholder="Search the docs..">
+    <div class="icons">
```
Comment (alexcracea) [inline index.html:14]: Lets enhance the semantics for this - Please use a `<ul>` with 4 `<li>` elements, and wrap an `<a>` around each svg icon, in the future we will make the click action to work correctly with javascript

File: index.html [hunk 6]
```diff
@@ -4,215 +4,54 @@
...
+        <li><a href="#background-box-2">Visual Style</a></li>
+        <li><a href="#background-box-3">Voice & Tone</a></li>
+        <li><a href="#background-box-4">Ul components</a></li>
+        <li><a href="#background-box-5">CSS Utility</a></li>
+        <li><a href="#background-box-6">Resources</a></li>
+      </ul>
+    </aside>
+    <content class="content-area">
+      <div class="section" id="background-box-1">
+        <span class="title-text">UI Components</span>
```
Comment (alexcracea) [inline index.html:42]: I see you want to have them categorised and its good, but if you want to continue with this implementation lets use a more BEM-named class, several examples: `section__category` or `section__eyebrow`

**Review by alexcracea — CHANGES_REQUESTED** [review]

File: scss/layout/_section.scss (hunk 1 above)
Comment (7ewen) [reply scss/layout/_section.scss:41]: Done

File: scss/layout/_section.scss (hunk 2 above)
Comment (7ewen) [reply scss/layout/_section.scss:52]: Done, but unfortunately I didn't see how github suggest it.

File: scss/layout/_section.scss (hunk 3 above)
Comment (7ewen) [reply scss/layout/_section.scss:23]: checked each line and changed, done.

File: scss/components/_header.scss (hunk 4 above)
Comment (7ewen) [reply scss/components/_header.scss:12]: changed to #mySearch {} alone

File: index.html (hunk 5 above)
Comment (7ewen) [reply index.html:14]: @alexcracea   That's the most dificult part, I tried to remove <a> cause it's showing clicking spaces between SVG's.  
Screenshot_14

screen attached. Also the measurements of SVG didn't change.
2nd screen attached.
measurments

idk why, please help! 🙏

**Pull request merged**